metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/configuration/metron-env.xml

```diff
@@ -253,69 +253,295 @@
+        <description>Indexing Topology JVM Options</description>
+        <value/>
+        <display-name>Indexing childopts</display-name>
+        <value-attributes>
+            <empty-value-valid>true</empty-value-valid>
+        </value-attributes>
+    </property>
+    <property>
+        <name>indexing_topology_max_spout_pending</name>
+        <description>Indexing Topology Spout Max Pending Tuples</description>
+        <value/>
+        <display-name>Indexing Max Pending</display-name>
+        <value-attributes>
+            <empty-value-valid>true</empty-value-valid>
+        </value-attributes>
+    </property>
+    <property>
+        <name>indexing_kafka_start</name>
+        <description>Indexing Topology Spout Offset</description>
+        <value>UNCOMMITTED_EARLIEST</value>
+        <display-name>Indexing Offset</display-name>
+        <value-attributes>
+            <type>value-list</type>
+            <entries>
+                <entry>
+                    <value>EARLIEST</value>
+                </entry>
+                <entry>
+                    <value>LATEST</value>
+                </entry>
+                <entry>
+                    <value>UNCOMMITTED_EARLIEST</value>
+                </entry>
+                <entry>
+                    <value>UNCOMMITTED_LATEST</value>
+                </entry>
+            </entries>
+            <selection-cardinality>1</selection-cardinality>
+        </value-attributes>
+    </property>
+    <property>
+        <name>indexing_input_topic</name>
+        <description>Indexing Input Topic</description>
+        <value>indexing</value>
+        <display-name>Indexing Input Topic</display-name>
+    </property>
+    <property>
+        <name>indexing_error_topic</name>
+        <description>Indexing Error Topic</description>
+        <value>indexing</value>
+        <display-name>Indexing Error Topic</display-name>
+    </property>
+    <property>
+        <name>indexing_writer_class_name</name>
+        <description>Indexing Writer Class Name</description>
+        <value>org.apache.metron.elasticsearch.writer.ElasticsearchWriter</value>
+        <display-name>Indexing Writer Class Name</display-name>
+    </property>
+    <property>
+        <name>bolt_hdfs_rotation_policy</name>
+        <description>Class Name of HDFS Rotation Policy</description>
+        <value>org.apache.storm.hdfs.bolt.rotation.TimedRotationPolicy</value>
+        <display-name>HDFS Rotation Policy</display-name>
+    </property>
+    <property>
+        <name>bolt_hdfs_rotation_policy_count</name>
+        <description>HDFS Rotation Policy Count</description>
+        <value>1</value>
+        <display-name>HDFS Rotation Policy Count</display-name>
+    </property>
+    <property>
+        <name>bolt_hdfs_rotation_policy_units</name>
+        <description>Frequency of HDFS File Rotation</description>
+        <value>DAYS</value>
+        <display-name>HDFS Rotation Policy Units</display-name>
```

Comment:
I think the values would be GB, KB, MB, TB  based on what[ I am reading here.](https://storm.apache.org/releases/1.0.1/javadocs/org/apache/storm/hdfs/bolt/rotation/FileSizeRotationPolicy.Units.html)